jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/TextAreaAssert.java

```diff
@@ -4,99 +4,306 @@
+        OutlinedAssert<TextAreaAssert, TextArea>, ReadOnlyAssert<TextAreaAssert, TextArea>,
+        RoundedAssert<TextAreaAssert, TextArea>, ShapedAssert<TextAreaAssert, TextArea>,
+        SingleLineAssert<TextAreaAssert, TextArea>, ThemeAssert<TextAreaAssert, TextArea> {
+    @Override
     @JDIAction("Assert that '{name}' text {0}")
     public TextAreaAssert text(Matcher<String> condition) {
-        jdiAssert(element().getText(), condition);
-        return this;
-    }
-
-    @JDIAction("Assert that '{name}' is filled")
-    public TextAreaAssert filled() {
-        jdiAssert(element().hasClass(FILLED_CLASS) ? "filled" : "not filled", Matchers.is("filled"));
-        return this;
-    }
-
-    @JDIAction("Assert that '{name}' is outlined")
-    public TextAreaAssert outlined() {
-        jdiAssert(element().hasClass(OUTLINED_CLASS) ? "outlined" : "not outlined", Matchers.is("outlined"));
+        String actualText = element().getText();
+        jdiAssert(actualText, condition, String.format("Actual element's text '%s' is not equal to expected " +
+                "'%s'", actualText, condition));
         return this;
     }
 
     @JDIAction("Assert that '{name}' is autoGrow")
     public TextAreaAssert autoGrow() {
-        jdiAssert(element().hasClass(AUTO_GROW_CLASS) ? "auto grow" : "not auto grow", Matchers.is("auto grow"));
+        jdiAssert(element().isAutogrow(), Matchers.is(true), "Element is not autogrow");
         return this;
     }
 
-    @JDIAction("Assert that '{name}' is not resizable")
-    public TextAreaAssert notResizable() {
-        jdiAssert(element().hasClass(NO_RESIZE_CLASS) ? "not resizable" : "resizable", Matchers.is("not resizable"));
+    @JDIAction("Assert that '{name}' is not autoGrow")
+    public TextAreaAssert notAutoGrow() {
+        jdiAssert(element().isAutogrow(), Matchers.is(false), "Element is autogrow");
         return this;
     }
 
-    @JDIAction("Assert that '{name}' height {0}")
-    public TextAreaAssert height(Matcher<Integer> condition) {
-        jdiAssert(element().height(), condition);
+    @JDIAction("Assert that '{name}' is not resizable")
+    public TextAreaAssert notResizable() {
+        jdiAssert(element().isNotResizable(), Matchers.is(true), "Element is resizable");
         return this;
     }
 
-    @JDIAction("Assert that '{name}' background-color {0}")
-    public TextAreaAssert backgroundColor(String color) {
-        jdiAssert(element().backgroundColor(), equalTo(color));
+    @JDIAction("Assert that '{name}' is not resizable")
+    public TextAreaAssert resizable() {
+        jdiAssert(element().isNotResizable(), Matchers.is(false), "Element is not resizable");
         return this;
     }
 
-    public TextAreaAssert backgroundColor(Enum<?> color) {
-        backgroundColor(color.toString());
+    @JDIAction("Assert that '{name}' height '{0}'")
+    public TextAreaAssert height(Matcher<Integer> condition) {
```

Comment:
For what case you add this method? 
We've already have methods to compare/less/more